sections/semantics-embedded-content.include

```diff
@@ -12105,40 +12107,33 @@ red:89
   <dl class="domintro">
 
     <dt><var>map</var> . <code>areas</code></dt>
-
     <dd>
-
-    Returns an <code>HTMLCollection</code> of the <{area}> elements in the
-    <{map}>.
-
+      Returns an <code>HTMLCollection</code> of the <{area}> elements in the <{map}>.
     </dd>
 
     <dt><var>map</var> . <code>images</code></dt>
-
     <dd>
-
-    Returns an <code>HTMLCollection</code> of the <code>img</code> and <code>object</code>
-    elements that use the <{map}>.
-
+      Returns an <code>HTMLCollection</code> of the <code>img</code> and <code>object</code>
+      elements that use the <{map}>.
     </dd>
 
   </dl>
 
-  The <dfn attribute for="HTMLMapElement"><code>areas</code></dfn> attribute must return an
+  The <dfn attribute for="HTMLMapElement"><code>areas</code></dfn> IDL attribute must return an
   <code>HTMLCollection</code> rooted at the <{map}> element, whose filter matches only
   <{area}> elements.
 
-  The <dfn attribute for="HTMLMapElement"><code>images</code></dfn> attribute must return an
+  The <dfn attribute for="HTMLMapElement"><code>images</code></dfn> IDL attribute must return an
   <code>HTMLCollection</code> rooted at the {{Document}} node, whose filter matches only
-  <code>img</code> and <{object}> elements that are associated with this <code>map</code>
-  element according to the <a>image map</a> processing model.
+  <{img}> and <{object}> elements that are associated with this <{map}> element according
```

Comment:
From memory, `object` cannot have a `map` any more (few browsers implemented that properly, and they are not "modern" ones), and it is not listed as an allowed attribute. Please check and adjust accordingly.